src/go/cmd/token-vendor/README.md

```diff
@@ -54,6 +54,21 @@ refuse eg. robots to register other robots.
 * Body: application/x-pem-file
 * Response: only http status code
 
+### /public-key.configure: Customize robot registration
```

Comment:
Doing that only in retro-perspective sounds like there is a race-condition where for a short-time the robot-service account is used. Can we also add this at registering?

Reply:
It will happen when you bind the IPC

Reply:
I think we should also try to add it to the publish call.